recipes/pyside2/yum_requirements.txt

```diff
@@ -0,0 +1,9 @@
+libX11-devel
+libXt-devel
+libXext-devel
+libXrender-devel
+gtk2-devel
```

Comment:
All but gtk are available as conda packages now. (And hopefully gtk will be available soon as well.)
Could you try using those? (No need to do it here, that experiment can happen on the feedstock.)

Reply:
ok I'll look into this, thanks